packages/xsnap/src/xsnap.js

```diff
@@ -132,14 +136,14 @@ export function xsnap(options) {
 
   /**
    * @param {string} code
-   * @returns {Promise<void>}
+   * @returns {Promise<Uint8Array | void>}
```

Comment:
This should be unconditionally `Promise<Uint8Array>` now. The caller should be able to rely on the fully defined type.